package-avrdude.bash

```diff
@@ -93,7 +93,6 @@ else
   rm -f avrdude-${OUTPUT_VERSION}-${OUTPUT_TAG}.tar.bz2
   cp -a objdir avrdude
   tar -cjvf avrdude-${OUTPUT_VERSION}-${OUTPUT_TAG}.tar.bz2 avrdude
-  rm -r avrdude
+#  rm -r avrdude
```

Comment:
Was this change intentional? It doesn't seem related to the stated purpose of the PR.

Reply:
Nope.

Reply:
Fixed now.